packages/developer-extension/src/background/store.ts

```diff
@@ -0,0 +1,30 @@
+import { Store } from '../common/types'
+import { listenAction, sendAction } from './actions'
+
+export const store: Store = {
+  devServerStatus: 'checking',
+  logEvents: true,
+  useDevBundles: false,
+  useRumRecorder: false,
+}
+
+export function setStore(newStore: Partial<Store>) {
+  if (wouldModifyStore(newStore)) {
+    Object.assign(store, newStore)
+    sendAction('newStore', store)
+    chrome.storage.local.set({ store })
```

Comment:
Having a single store for all tabs could lead to extension state displayed nnot being consistent with tab state.
What about managing the state of each tab separately?

Reply:
Sure, I designed the state of the extension to be global, not local to the tab. But I can change this.

Reply:
So, I'd like to keep this as it is for now: I like to have a persistent configuration (accross browser restarts), and using tab ids to store the state won't allow this.

In the future we may reconsider this. It could be nice to use a devtools panel to display SDK related stuff and extension configuration. In this case, I think it would make sense to apply the extension when the devtools are open, and have its effect apply only on the related tab.

Reply:
sure